user-guide/Advanced_Modules/SRM/SRM_Quarantine.md

```diff
@@ -0,0 +1,438 @@
+            foreach (var trigger in quarantinedUsage.QuarantineTriggers)
+            {
+                errorBuilder.Append($"\t\t{trigger.QuarantineReason}");
+                errorBuilder.AppendLine();
+            }
+        }
+    }
+
+    // Example value of 'errorBuilder.ToString()':
+    // Scheduling conflict. The following bookings need to be moved to quarantine::
+    //     Test Booking 1 (54add931-66fc-44f5-a76e-95ad0317f6af):
+    // 	       resource with ID 'a250cffb-7054-4704-aa58-96200b0c49b3' will be quarantined because:
+    // 		       ReservationInstanceUpdated
+}
+```
+
+When a booking is saved with the *forceQuarantine* flag set to *true*, there will be warnings about the bookings that have automatically been moved to quarantine.
+
+```csharp
+// Save a 'ReservationInstance' and force quarantine
+helper.AddOrUpdateReservationInstances(forceQuarantine: true, reservation);
+
+var traceData = helper.GetTraceDataLastCall();
+
+// When forcing quarantine the call should succeed even if there are conflicts.
+// If the traceData does not indicate the call succeeded the error must be handled.
+if (!traceData.HasSucceeded())
+{
+    // The script should handle the error here, but in this example we will just return.
+    return;
+}
+
+foreach (var warning in traceData.WarningData)
+{
+    if (!(warning is ResourceManagerWarningData resourceManagerWarning) || 
+        resourceManagerWarning.WarningReason != ResourceManagerWarningData.Reason.ReservationInstancesMovedToQuarantine)
+    {
+        // For the sake of example we only handle the quarantine warning here, but if there is a different warning
+        // the script should of course not ignore it but handle it appropriately.
+        continue;
+    }
+
+    var bookingIds = string.Join(", ", resourceManagerWarning.SubjectIds);
+    var warningString = "The following bookings were moved to quarantine: {bookingIds}";
+    
+    // Example value of 'warningString': 
+    // The following bookings were moved to quarantine: 54add931-66fc-44f5-a76e-95ad0317f6af, 69958b9b-5704-4532-bb08-b021c5437084
+}
+```
+
+### Quarantine on resource updates
+
+Quarantine on resource updates works almost the same as quarantine on booking updates. Similar as for booking updates, bookings will only be moved to quarantine if the *forceQuarantine* flag is passed.
+
+The following updates to resources can trigger quarantine:
+
+1. Decreasing the *MaxConcurrency* of a resource.
+2. Changing the mode of a resource to *Maintenance*. This will move all bookings using that resource to quarantine.
+3. Deleting a capacity or capability. All bookings using the deleted capacity/capability of this resource will be moved to quarantine.
+4. Decreasing a maximum capacity in such a way that existing bookings are no longer supported.
+5. Changing or removing a capability in such a way that currently booked values are no longer supported.
```

Comment:
Is the order of this list relevant? If not, it would be better to change this to a bulleted list instead of a numbered list. If the order is relevant, just "1." should be used for each item.

Reply:
The order is not relevant here, changed to a bulleted list.